tasks/main.yml

```diff
@@ -56,6 +33,14 @@
     group: "{{ ldaptoolbox_openldap_configuration_group }}"
     mode: "{{ ldaptoolbox_openldap_configuration_mode }}"
 
+- name: deploy sladp-cli file
```

Comment:
I have kept this in rpm branch, but it's a pity we have to maintain a whole new configuration file here.
It would have been simpler to change the few modification via a `lineinfile` task...